server/service/src/sync/translations/name.rs

```diff
@@ -136,6 +137,7 @@ pub struct LegacyNameRow {
     #[serde(deserialize_with = "zero_date_as_option")]
     #[serde(serialize_with = "date_option_to_isostring")]
     pub date_of_death: Option<NaiveDate>,
+    #[serde(deserialize_with = "object_fields_as_option")]
```

Comment:
Handles generic Value here also to turn `""` into `None`

Reply:
Lets take this one out as don't want to risk breaking the SAGE integration (low chance but still...)